<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="kitinfoButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Kit info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plantinfoButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Plant info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="userinfoButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>User info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>admin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
